framework/src/media/utils/MediaUtils.cpp

```diff
@@ -622,5 +622,64 @@ bool writeWavHeader(FILE *fp, unsigned int channel, unsigned int sampleRate, aud
 
 	return true;
 }
+
+static uint32_t bitcount(uint32_t x)
```

Comment:
We can use `__builtin_popcount` if we are using gcc.
How about this?
```
#ifdef __GNUC__
	cout << __builtin_popcount(x);
#else
   // your code
#endif
```

Reply:
I will try it, thanks

Reply:
Fixed, thanks